src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -994,8 +994,7 @@ public ServicesManagerMock getServicesManager()
 	@Override
 	public World createWorld(WorldCreator creator)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return addSimpleWorld(creator.name());
```

Comment:
The point of a `WorldCreator` is to create a World that fits certain requirements.
These requirements include:
* Name
* Seed
* Environment
* ChunkGenerator
* BiomeProvider
* WorldType
* GenerateStructures (boolean)
* GeneratorSettings
* HardcoreMode (boolean)

Your implementation only fulfills one of these requirements, if I want to create a Nether world and I get an overworld, clearly something's wrong.

If you are interested in a proper implementation of this, I suggest you to take a test-driven approach in this case.
Write a Test class for the `WorldCreator` and write test cases for each of these requirements.
Set one of those values, create a world and check if the requirement was fulfilled.
Then implement accordingly.

Of course we don't need actually need chunk generation or biomes and stuff here.
But environment, seed, worldtype and such should be the minimum here.
The rest can always be implemented later (if anyone cares for those features)

Reply:
I can add mocked Environment, but both getSeed and getWorldType return unimplemented exceptions, so does anyone even care about that?

Reply:
In terms of simple get operations, yes. Those should be covered.